Black-Market/Black-Market.xcodeproj/xcshareddata/xcschemes/Black-Market.xcscheme

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you revise this folder is correctly added to the gitignore?

Reply:
Done